server/@types/forms/index.d.ts

```diff
@@ -12,6 +12,6 @@ declare module 'forms' {
   }
   export interface DecisionEvidenceForm {
     evidenceText?: string
-    isSensitive?: boolean
+    isSensitive?: string
```

Comment:
oh so redis can't cope with booleans?

Reply:
...if so we should do the string->boolean conversion in the cacheService, not expose the type here

Reply:
yeah, `string | Buffer` only